deploy/stf-1.3/rhos-dashboard.yaml

```diff
@@ -2414,25 +2390,52 @@ spec:
       "tags": [],
       "templating": {
         "list": [
+          {
+            "allValue": null,
+            "current": {
+              "selected": true,
+              "text": "osp161",
+              "value": "osp161"
+            },
+            "datasource": null,
+            "definition": "label_values(collectd_cpu_percent, service)",
+            "description": null,
+            "error": null,
+            "hide": 0,
+            "includeAll": false,
+            "label": "cloud",
+            "multi": false,
+            "name": "clouds",
+            "options": [],
+            "query": {
+              "query": "label_values(collectd_cpu_percent, service)",
+              "refId": "StandardVariableQuery"
+            },
+            "refresh": 1,
+            "regex": "/.+-(.+)-coll-meter/",
+            "skipUrlSync": false,
+            "sort": 0,
+            "type": "query"
+          },
           {
             "allValue": null,
             "current": {
               "selected": false,
-              "text": "ceph-0.osp-cloudops-01",
-              "value": "ceph-0.osp-cloudops-01"
+              "text": "ceph-0-16-1.redhat.local",
+              "value": "ceph-0-16-1.redhat.local"
```

Comment:
Do we need to keep these defaults in here?

Reply:
nope